### DISCUSSION TITLE
Error: unknown field `lazy`, expected `resolveFully`

### DISCUSSION THREAD

**Discussion by DuCanhGH** [DuCanhGH/next-pwa#93] [Announcements]
If you see the error above when you use `next-pwa` with a custom worker, `PluginOptions.fallbacks`, or `PluginOptions.cacheOnFrontEndNav`, please run the following:
```bash
# if you use npm
npm i -D @swc/core@1.3.78
# if you use yarn
yarn add -D @swc/core@1.3.78
# if you use pnpm
pnpm add -D @swc/core@1.3.78
# if you use bun
bun add -D @swc/core@1.3.78
```
This issue seems to have been happening since `@swc/core@1.3.80`, so yeah. Removing `module.lazy` and `module.importInterop` from our `swcConfig` works, but they don't seem to have been removed, so this is most likely a bug.

**Comment by maiconcarraro** [comment]
@DuCanhGH in [this PR](https://github.com/swc-project/swc/commit/7e8d72d03b312b7a48c17afa8d2a4d7f4e802a6a#diff-05ec577011f6096aac154c266838a96b40e857713f31a3441fc73baf1d90580eR1576), all changes for es6 configuration removed the extra options, the awkward situation is for nodenext configuration, I shared a comment [here ](https://github.com/swc-project/swc/issues/7979#issuecomment-1728806816) trying to help

**Reply by DuCanhGH** [reply]
Thanks!

**Reply by maiconcarraro** [reply]
also just tested using your new release + latest next, all good! ty <3
